console/account/reference-content/scaleway-network-information.mdx

```diff
@@ -91,16 +85,23 @@ IPv6:
   are deprecated and should no longer be used.
 </Message>
 
+## Monitoring of Dedibox servers
+
+Our monitoring servers are located in the IP subnet `62.210.16.0/24`.
+
 ## NTP server
```

Comment:
If all the rest is still DDX-specific, maybe they should be under H3s (and a 'Dedibox" H2)?